router/router.go

```diff
@@ -1849,3 +1860,66 @@ func (rt *HandleT) Resume() {
 
 	rt.paused = false
 }
+
+// Currently the retry logic has been implemented for only OAuth
+func (rt *HandleT) SendToTransformerProxyWithRetry(val integrations.PostParametersT,
```

Comment:
why do we need to pass accessToken as an explicit parameter when it is already part of PostParametersT